tests/integration/test_rex.py

```diff
@@ -334,55 +333,56 @@ def test_math_operations(c, df):
             , ACOS(b) AS "acos"
             , ASIN(b) AS "asin"
             , ATAN(b) AS "atan"
-            , ATAN2(a, b) AS "atan2"
-            , CBRT(b) AS "cbrt"
```

Comment:
We should be able to comment these out using [SQL comments](https://www.w3schools.com/Sql/sql_comments.asp), so with something like:

```sql
            -- , ATAN2(a, b) AS "atan2"
            -- , CBRT(b) AS "cbrt"
```

Reply:
Good to know, thanks! I'll update it right now.